src/data/services/LmsApiService.js

```diff
@@ -219,6 +228,10 @@ class LmsApiService {
     return LmsApiService.apiClient().post(LmsApiService.notificationReadUrl, formData);
   }
 
+  static fetchEnterpriseCustomerIntegrationConfigs(options) {
+    return LmsApiService.apiClient().get(`${LmsApiService.lmsIntegrationUrl}/configs/?${qs.stringify(options)}`);
```

Comment:
i did a little digging to see if we could remove this 3rd party dependency on `query-string` but we cannot. i am kind of bummed there isn't an easier, built-in way to turn a dictionary into a query string, and you've gotta use this 3rd party lib for basically just this.